src/detection/detection-wrapper.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react'
+import { Grid } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useStyles } from 'detection/style'
+
+export const DetectionWrapper = ({ children }) => <Grid container justify={'center'} alignContent={'center'}
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125618434-6442128e-cf09-4a6b-9867-4806735d8a70.png)
Since we're already in the `/detection` directory, all these `detection-omething` naming s are unnecessary. Let's remove the `detection` prefix.

Reply:
done